### PR TITLE
Tetris Configurator support and Rules update

### PR DESCRIPTION
**Pull request by noroadsleft**
+`info.json`
+`LAYOUTS = planck_mit` &#x2192; `rules.mk`

### CODE REVIEW

File: keyboards/tetris/rules.mk [hunk 1]
```diff
@@ -55,4 +55,6 @@ BACKLIGHT_ENABLE ?= no  # Enable keyboard backlight functionality
 AUDIO_ENABLE ?= yes
 RGBLIGHT_ENABLE ?= yes
 TAP_DANCE_ENABLE = yes
```
Comment (drashna) [inline keyboards/tetris/rules.mk]: Tap Dance should be disabled by default, and should only be enabled for keymaps that need it. 

This is why it's erroring out in Travis.

Comment (noroadsleft) [reply keyboards/tetris/rules.mk]: So disable Tap Dance by

1. change `yes` to `no` here
2. add a `keymaps/default/config.h` file containing `#define TAPPING_TERM 200`
  - The default keymap uses tap dance

Is that correct?

**Comment by noroadsleft** [comment]
Yeah... that achieved nothing. :(

**Comment by drashna** [comment]
Wrap the Tap Dance stuff in the keymap with ifdef's.  Eg: 
```c
#ifdef TAP_DANCE_ENABLE
qk_tap_dance_action_t tap_dance_actions[] = {
  [TD_Mute_Next]  = ACTION_TAP_DANCE_DOUBLE( KC_MUTE, KC_MNXT )
  ,[TD_SFT]  = ACTION_TAP_DANCE_FN_ADVANCED( NULL, dance_cln_finished, dance_cln_reset )
  ,[TD_LBRC]  = ACTION_TAP_DANCE_DOUBLE( KC_LBRC,   KC_LCBR )
  ,[TD_RBRC]  = ACTION_TAP_DANCE_DOUBLE( KC_RBRC,   KC_RCBR )
};
#endif //TAP_DANCE_ENABLE
```

**Comment by noroadsleft** [comment]
Should I replace the Tap Dance keycodes in the Base layer, or leave them?

**Comment by drashna** [comment]
hmmm. 

I think tapdance needs to be enabled for them to work.  And if that's the case, then it may be best to add a rules.mk for the default keymap, and leave it at that.

**Comment by drashna** [comment]
Awesome, thanks!

**Pull request merged**